### PR TITLE
Created a schema based on a diagram

### PR DESCRIPTION
**Pull request by alonsofl**
### Summary
Implemented the SQL needed to build a database based on a diagram.

#### Details
- Implemented the needed SQL to create the requested database.
- Created a `schema_based_on_diagram.sql` file.
- Added required `Foreign Keys`
- Created a join table for the many-to-many relationships.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  quantity INT,
+  total_price DECIMAL,
+  invoice_id INT REFERENCES invoices(id),
+  treatment_id INT REFERENCES treatments(id)
+);
+
+CREATE TABLE histories_treatments (
+  treatment_id INT REFERENCES treatments(id),
+  medical_history_id INT REFERENCES medical_histories(id)
+);
```
Comment (arslanbisharat) [inline schema_based_on_diagram.sql:42]: - [ ] Let's make sure all FOREIGN KEYS have indexes created as requested in the guidelines. 
Capture

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (jmontanholi) [reply schema_based_on_diagram.sql:42]: Sorry @arslanbisharat what does this mean exactly? Do we need to use the `CREATE INDEX` command?

**Comment by Davidosky007** [comment]
## Hi @alonsofl @jmontanholi 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
job-welldone-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**